### PR TITLE
Create netsec-spheres-of-support-for-dedicated-and-managed-ops.md

### PR DESCRIPTION
**Pull request by abekele0413**
Initial commit for netsec-spheres-of-support-for-dedicated-and-managed-ops.md document

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-567707e320/how-to/).

File: content/dedicated-hosting/netsec-spheres-of-support-for-dedicated-and-managed-ops.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+created_by: NetSec Standardization Group
+last_modified_date: '2019-02-25'
+last_modified_by: Adrian Bekele
+product: Dedicated Hosting
+product_url: dedicated-hosting
+---
+
+The Spheres of Support lists the technologies and features in the Network Security domain and defines support level for each item. 
+ 
+### Firewall support:
```
Comment (catlook) [inline content/dedicated-hosting/netsec-spheres-of-support-for-dedicated-and-managed-ops.md]: Add an introductory line before the list similar to: 

Firewall support contains the following elements:

File: content/dedicated-hosting/netsec-spheres-of-support-for-dedicated-and-managed-ops.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+* Customized patching, rollback capabilities, and iOS Updates
+* Problem, incident, and change management
+* Configuration changes on behalf of the customer
+* Performance monitoring and tuning
+* Configuration and support of IPsec-compliant VPNs
+* Configuration and maintenance of firewall policies to restrict access to environments
+* Provision VLANs and private IP space
+* Managed VPN access
+* SSL certificate installation and troubleshooting
+
```
Comment (catlook) [inline content/dedicated-hosting/netsec-spheres-of-support-for-dedicated-and-managed-ops.md]: Add an introductory line before this table like:

The following table shows which features are included in Cisco and Juniper firewalls:

File: content/dedicated-hosting/netsec-spheres-of-support-for-dedicated-and-managed-ops.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+| **High Availability (HA)** |  Active/Standby (stateful and non-stateful) (clustering - SRX)<Br/>ASA 5510 and above, ASA-X 5508, 5515 and above |✓|✓|
+| **Modes and modules** | Mode - Multi-Context Routed |✓||
+||Mode - Routed |✓| ✓|
+| **RackConnect** | RackConnect VLANs termination |✓| |
+| **RackConnect Global** |  RackConnect Global Classic Static |✓| ✓|
+||RackConnect Global 2.0 Static - BGP |✓| |
+||RackConnect Global 2.1 Static - BGP (with TOR)  |✓| |
+
+
+### Load balancer support:
```
Comment (catlook) [inline content/dedicated-hosting/netsec-spheres-of-support-for-dedicated-and-managed-ops.md]: add intro sentence

File: content/dedicated-hosting/netsec-spheres-of-support-for-dedicated-and-managed-ops.md [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+* Monitoring and management of Load balancers
+* Problem, incident, and change management
+* Configuration changes on behalf of the customer
+* Performance monitoring and tuning
+
```
Comment (catlook) [inline content/dedicated-hosting/netsec-spheres-of-support-for-dedicated-and-managed-ops.md]: add intro sentence

File: content/dedicated-hosting/netsec-spheres-of-support-for-dedicated-and-managed-ops.md [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+| **RackConnect Global** |  RackConnect Global Classic Static |✓(AFM only)||||
+| **SSL VPNs** |  SSL Client VPN on edge Big-IPs using APM module |✓||||
+| **SNI** | Server Name Indication  |✓|✓|✓|✓|
+
+^ VPX - Product in early access soon to be limited access
+
+^^ MPX - Product not launched, only supportability planned at this stage.
+  
+### Global load balancer
+
```
Comment (catlook) [inline content/dedicated-hosting/netsec-spheres-of-support-for-dedicated-and-managed-ops.md]: add intro sentence

**Comment by abekele0413** [comment]
changes have been made and re-submitted on PR#3209

**Pull request closed without merging**